core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -184,7 +184,10 @@ object Boot extends App with LazyLogging {
       gcsConfig.getInt("groupMonitor.workerCount"),
       userServiceConstructor))
 
-    BootMonitors.restartMonitors(slickDataSource, gcsDAO, submissionSupervisor, bucketDeletionMonitor)
+    if(sys.env.getOrElse("BACK_RAWLS", "false").toBoolean) {
```

Comment:
my assumption here is that when jenkins is deploying multiple instances of rawls it will pick one to be back-rawls and set an env var `BACK_RAWLS = true`

Reply:
you should read from config, if we decide to use env vars or system properties later we can still feed them in through config... let's keep all our config in the same spot